src/test/kotlin/sh/hsp/techtree/graphviz/DSLConverterTest.kt

```diff
@@ -39,11 +36,24 @@ class DSLConverterTest {
             )
         )
 
-        val converter = DSLConverter()
-
-        val result = converter.convert(model)
+        val result = sut.convert(model)
 
         "digraph { graph [splines=\"ortho\"] node [shape=\"box\"] edge [dir=\"back\"] nodeA nodeA -> nodeB nodeA -> nodeC nodeB nodeC }".shouldBe(result)
     }
 
+    @Test
+    fun givenSpacesInTitleShouldRegisterNodesInCorrectGraphvizDSL() {
+        val model = TreeModel(
+            listOf(
+                TreeNode("Big dog", listOf("Small dog")),
+                TreeNode("Small dog"),
+                TreeNode("Big cat")
+            )
+        )
+
+        val result = sut.convert(model)
+
+        "digraph { graph [splines=\"ortho\"] node [shape=\"box\"] edge [dir=\"back\"] \"Big dog\" \"Big dog\" -> \"Small dog\" \"Small dog\" \"Big cat\" }".shouldBe(result)
```

Comment:
Trzeba pomyśleć nad lepszym testowaniem tych grafów, bo jak się rozrosną to się zachlastam

Reply:
Ten kod bez enterów jest straszny xD
Nie widzę co jest tutaj nodem, a co edgem xD
Warto byłoby to ustrukturyzować.